web/src/test/java/org/cbioportal/web/GenesetDataControllerTest.java

```diff
@@ -64,46 +64,77 @@ public void setUp() throws Exception {
     }
 
     @Test
-    public void fetchGeneticDataItems() throws Exception {
+    public void fetchGeneticDataItemsWithSampleIds() throws Exception {
+        List<GenesetMolecularData> genesetDataItems = createGenesetDataItemsList();
+
+        String geneticProfileId = PROF_ID;
+        List<String> sampleIds = Arrays.asList(SAMPLE_STABLE_ID_1);
+        List<String> genesetIds = Arrays.asList(GENESET_ID_1, GENESET_ID_2);
+
+        // Argument values don't matter, just shows which fetchGenesetData is called
+        Mockito.when(genesetDataService.fetchGenesetData(
+            geneticProfileId,
+            sampleIds,
+            genesetIds)
+        ).thenReturn(genesetDataItems);
 
+        // Set sampleIds to ensure the fetchGenesetData variant above is called:
```

Comment:
I don't think this comment says anything useful. Trivial comments get ignored, and eventually age to the point that they're confusing or incorrect. I'd delete it.